api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -232,7 +246,7 @@ private List<GlobalProperty> filterGlobalPropertiesByViewPrivilege(List<GlobalPr
 	@Override
 	@Transactional(readOnly = true)
 	public String getGlobalProperty(String propertyName, String defaultValue) throws APIException {
-		String s = Context.getAdministrationService().getGlobalProperty(propertyName);
+		String s = getGlobalProperty(propertyName);
```

Comment:
It should be self.getGlobalProperty(propertyName). The reason is that if you just call getGlobalProperty, then no authorization or logging AOPs are applied.
If you see Context.get...Service within the same service you can assume that you need to call it via self.... to have all AOPs.